### DISCUSSION TITLE
Changing Theme

### DISCUSSION THREAD

**Discussion by itz-Amethyst** [Matt-FTW/dotfiles#22] [Q&A] [answered]
Hi i hope you're doing well
I've been using your dotfiles recently and just wanted to give a quick shoutout it's a great configuration.
However, i’ve noticed theme options feel somewhat limited, which might personally be a drawback.

While browsing through the files, I noticed that unfortunately there doesn’t seem to be an easy way to change the current theme and have it apply across most system sections.

I'm encountering an error when trying to open the Rofi theme selector. I’m not sure if this is a bug or an issue on my end. Do you have any suggestions that can help me to change the theme?

![image](https://github.com/user-attachments/assets/dfbdad9b-e8f3-4404-a5d0-3a9bf467ce70)

**Comment by Matt-FTW** [comment, marked as answer]
Hey, 
Thanks for the kind words buddy!

The issue has been fixed with f186141. Let me know if it still happens.

>However, i’ve noticed theme options feel somewhat limited, which might personally be a drawback.
> While browsing through the files, I noticed that unfortunately there doesn’t seem to be an easy way to change the current theme and have it apply across most system sections.

Yeah thats something I talk [over here](https://github.com/Matt-FTW/dotfiles/blob/main/.github/FAQ.md#are-custom-themes-something-youd-introduce-in-the-future).

It's a lot of work to refactorize and I dont have the time or the will to do it. Im open to PRs tho. Sorry about it!

**Reply by itz-Amethyst** [reply]
I'm glad you fixed it.

Can't wait for this feature to be released.
